docs/source/guide/configuration.rst

```diff
@@ -19,12 +19,14 @@ For details about credential configuration, see the :ref:`guide_credentials` gui
 
 
 Using the Config object
--------------------------
+-----------------------
 This option is for configuring client-specific configurations that affect the behavior of your specific client object only. As described earlier, there are options used here that will supersede those found in other configuration locations:
 
 * ``region_name`` (string) - The AWS Region used in instantiating the client. If used, this takes precedence over environment variable and configuration file values. But it doesn't overwrite a ``region_name`` value *explicitly* passed to individual service methods.
 * ``signature_version`` (string) - The signature version used when signing requests. Note that the default version is Signature Version 4. If you're using a presigned URL with an expiry of greater than 7 days, you should specify Signature Version 2.
-* ``s3`` (related configurations; dictionary) - Amazon S3 service-specific configurations. For more information, see the `Config reference <https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html>`_.
+* ``s3`` (related configurations; dictionary) - Amazon S3 service-specific configurations. For more information, see the `Botocore config reference <https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html>`_.
+* ``proxies`` (dictionary) - A dictionary mapping protocol names to the proxy servers that should be used when communicating with them. These proxies, if specified, are used for every request. See :ref:`specify_proxies` for more information.
```

Comment:
Is this accurate?

A dictionary that maps protocols to proxy servers so that Boto3 can look up the correct protocol to communicate with a server for each request. See ...

Reply:
To compensate for a couple of issues I see with the suggestion, I have changed the text to:

```
``proxies`` (dictionary) - Each entry maps a protocol name to the proxy server Boto3
should use to communicate using that protocol. See :ref:`specify_proxies` for more
information.
```